_dashboards/discover/multi-data-sources.md

```diff
@@ -41,22 +41,45 @@ To create a new data source connection:
 
 1. Open Dashboards. If you’re not running the security plugin, go to [`http://localhost:5601`](http://localhost:5601/). If you’re running the security plugin, go to [`https://localhost:5601`](https://localhost:5601/) and log in with the username `admin` and password `admin`.
 
-1. In the Dashboards console, choose **Stack Management** > **Data Sources `Experimental`** > **Data Sources** > **Create data source connection**, as shown in the following image.
+1. In the Dashboards console, navigate to **Stack Management** > **Data Sources `Experimental`** > **Data Sources**, and then select **Create data source connection**, as shown in the following image.
```

Comment:
```suggestion
1. In the Dashboards console, under **Management**, choose **Stack Management**. Then choose **Data Sources `Experimental`**, then choose **Data Sources**, and then choose **Create data source connection**, as shown in the following image.
```

Reply:
Good comment. I want to avoid repeating "choose" for each selection item in the UI. Otherwise, I've changed the suggested language. Let's see what editorial suggests.